crates/lib3h_protocol/src/types.rs

```diff
@@ -43,3 +43,89 @@ impl SpaceHash {
         SpaceHash(HashString::new())
     }
 }
+
+//--------------------------------------------------------------------------------------------------
+// EntryHash: newtype for HashString
+//--------------------------------------------------------------------------------------------------
+
+#[derive(PartialOrd, PartialEq, Eq, Ord, Clone, Debug, Serialize, Deserialize, Default, Hash)]
+pub struct EntryHash(HashString);
```

Comment:
Can you also do Shrinkwrap and shrinkwrap(mutable) a'la https://github.com/holochain/lib3h/blob/develop/crates/zombie_actor/src/lib.rs#L22-L23 so we get AsRef and friends?

Reply:
shrinkwrap(mutable) doesnt work as HashString would need to derive it also I believe.
Don't see the need anyway as there is no need to mutate hashes, no?